tools/converter/mule2ballerina/src/main/java/org/wso2/ei/tools/mule2ballerina/visitor/TreeVisitor.java

```diff
@@ -145,7 +157,10 @@ public void visit(Payload payload) {
             switch (mimeType) {
             case XML:
                 ballerinaASTAPI.addTypes(Constant.BLANG_TYPE_XML); //type of the variable
-                ballerinaASTAPI.createBackTickExpression(Constant.BACKTICK + payload.getValue() + Constant.BACKTICK);
+                /*Backtick expression is not longer supported in Ballerina. Improvements may come in a future release*/
+                // ballerinaASTAPI.createBackTickExpression(Constant.BACKTICK + payload.getValue() + Constant.BACKTICK);
```

Comment:
let's remove commented code

Reply:
This was fixed in this commit >> https://github.com/wso2/product-ei/pull/910/files#diff-a7c35609bfca9af1252baf6d499e26feR287